src/rpc/net.cpp

```diff
@@ -544,6 +544,7 @@ static UniValue getnetworkinfo(const JSONRPCRequest& request)
         obj.pushKV("connections", conn_counts.all);
         obj.pushKV("connections_in", conn_counts.in);
         obj.pushKV("connections_out", conn_counts.out);
+        obj.pushKV("connections_tor_only", conn_counts.tor_only);
```

Comment:
`connections_tor_only` should be documented in the `RPCResult` above

Reply:
Also, the RPC and net changes are not gui related, so should go into the main repo

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/20172.